ERROR - 2024-10-26 10:33:44 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\codeigniter_pjnew\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\codeigniter_pjnew\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'codeigniter', 3306, '', 0)
#1 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\codeigniter_pjnew\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter_pjnew\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter_pjnew\app\Controllers\EmployeeController.php(9): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(933): App\Controllers\EmployeeController->Index()
#8 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmployeeController))
#9 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter_pjnew\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\codeigniter_pjnew\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\codeigniter_pjnew\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\codeigniter_pjnew\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\codeigniter_pjnew\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\codeigniter_pjnew\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\codeigniter_pjnew\app\Controllers\EmployeeController.php(9): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(933): App\Controllers\EmployeeController->Index()
#7 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmployeeController))
#8 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codeigniter_pjnew\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\codeigniter_pjnew\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\codeigniter_pjnew\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-10-26 10:33:44 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
[Method: GET, Route: employee]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `employees`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\EmployeeController.php(9): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EmployeeController->Index()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmployeeController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-10-26 10:34:44 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\codeigniter_pjnew\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\codeigniter_pjnew\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'codeigniter', 3306, '', 0)
#1 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\codeigniter_pjnew\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter_pjnew\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter_pjnew\app\Controllers\EmployeeController.php(9): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(933): App\Controllers\EmployeeController->Index()
#8 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmployeeController))
#9 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter_pjnew\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\codeigniter_pjnew\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\codeigniter_pjnew\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\codeigniter_pjnew\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\codeigniter_pjnew\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\codeigniter_pjnew\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\codeigniter_pjnew\app\Controllers\EmployeeController.php(9): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(933): App\Controllers\EmployeeController->Index()
#7 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmployeeController))
#8 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codeigniter_pjnew\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\codeigniter_pjnew\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\codeigniter_pjnew\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-10-26 10:34:44 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
[Method: GET, Route: employee]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `employees`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\EmployeeController.php(9): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EmployeeController->Index()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmployeeController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-10-26 10:34:46 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\codeigniter_pjnew\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\codeigniter_pjnew\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'codeigniter', 3306, '', 0)
#1 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\codeigniter_pjnew\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter_pjnew\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter_pjnew\app\Controllers\EmployeeController.php(9): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(933): App\Controllers\EmployeeController->Index()
#8 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmployeeController))
#9 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter_pjnew\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\codeigniter_pjnew\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\codeigniter_pjnew\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\codeigniter_pjnew\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\codeigniter_pjnew\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\codeigniter_pjnew\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\codeigniter_pjnew\app\Controllers\EmployeeController.php(9): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(933): App\Controllers\EmployeeController->Index()
#7 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmployeeController))
#8 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codeigniter_pjnew\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\codeigniter_pjnew\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\codeigniter_pjnew\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-10-26 10:34:46 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
[Method: GET, Route: employee]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `employees`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\EmployeeController.php(9): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EmployeeController->Index()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmployeeController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-10-26 10:36:59 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\codeigniter_pjnew\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\codeigniter_pjnew\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'codeigniter', 3306, '', 0)
#1 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\codeigniter_pjnew\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter_pjnew\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter_pjnew\app\Controllers\EmployeeController.php(9): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(933): App\Controllers\EmployeeController->Index()
#8 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmployeeController))
#9 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter_pjnew\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\codeigniter_pjnew\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\codeigniter_pjnew\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\codeigniter_pjnew\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\codeigniter_pjnew\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\codeigniter_pjnew\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\codeigniter_pjnew\app\Controllers\EmployeeController.php(9): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(933): App\Controllers\EmployeeController->Index()
#7 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmployeeController))
#8 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codeigniter_pjnew\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\codeigniter_pjnew\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\codeigniter_pjnew\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-10-26 10:36:59 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
[Method: GET, Route: employee]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `employees`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\EmployeeController.php(9): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EmployeeController->Index()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmployeeController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-10-26 10:37:01 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\codeigniter_pjnew\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\codeigniter_pjnew\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'codeigniter', 3306, '', 0)
#1 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\codeigniter_pjnew\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter_pjnew\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter_pjnew\app\Controllers\EmployeeController.php(9): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(933): App\Controllers\EmployeeController->Index()
#8 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmployeeController))
#9 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter_pjnew\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\codeigniter_pjnew\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\codeigniter_pjnew\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\codeigniter_pjnew\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\codeigniter_pjnew\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\codeigniter_pjnew\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\codeigniter_pjnew\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\codeigniter_pjnew\app\Controllers\EmployeeController.php(9): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(933): App\Controllers\EmployeeController->Index()
#7 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmployeeController))
#8 C:\xampp\htdocs\codeigniter_pjnew\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codeigniter_pjnew\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\codeigniter_pjnew\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\codeigniter_pjnew\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2024-10-26 10:37:01 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
[Method: GET, Route: employee]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `employees`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\EmployeeController.php(9): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EmployeeController->Index()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EmployeeController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-10-26 12:40:03 --> ParseError: syntax error, unexpected token "}", expecting ";"
[Method: GET, Route: employee]
in APPPATH\Controllers\EmployeeController.php on line 33.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\codeigniter_pjnew\\app\\Controllers\\EmployeeController.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\EmployeeController')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\EmployeeController')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\EmployeeController', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-10-26 12:46:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-26 12:47:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-26 12:50:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-26 12:50:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-26 12:53:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-26 12:53:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-26 13:11:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-26 13:22:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-10-26 13:23:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
